### PR TITLE
fix(tsc-files): import @testing-library/jest-dom/jest-globals types in jest setup files

### PR DESCRIPTION
**Pull request by calebpollman**
#### Description of changes
Fix type checking on commit

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/react-liveness/src/components/FaceLivenessDetector/shared/__tests__/FaceLivenessErrorModal.test.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ const {
 } = defaultErrorDisplayText;
 
 describe('FaceLivenessErrorModal', () => {
-  it('should render the component content appropriately', () => {
+  it('should render the component content sappropriately', () => {
```
Comment (esauerbo) [inline packages/react-liveness/src/components/FaceLivenessDetector/shared/__tests__/FaceLivenessErrorModal.test.tsx:24]: Lol was this intentional?

**Review by esauerbo — APPROVED** [review]
Left a comment but otherwise LGTM

File: packages/react-liveness/src/components/FaceLivenessDetector/shared/__tests__/FaceLivenessErrorModal.test.tsx (hunk 1 above)
Comment (calebpollman) [reply packages/react-liveness/src/components/FaceLivenessDetector/shared/__tests__/FaceLivenessErrorModal.test.tsx:24]: Whoops haha, it was intentional to force type checking on a commit. Meant to remove it

Comment (calebpollman) [reply packages/react-liveness/src/components/FaceLivenessDetector/shared/__tests__/FaceLivenessErrorModal.test.tsx:24]:
```suggestion
  it('should render the component content appropriately', () => {
```

**Review by esauerbo — APPROVED** [review]

**Review by dbanksdesign — APPROVED** [review]

**Review by zchenwei — APPROVED** [review]

**Pull request merged**